Update set-up-phone-system-voicemail.md (#2159)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Teams/set-up-phone-system-voicemail.md

## Setting voicemail policies in your organization

> [!WARNING]

> > For Skype for Business customers, disabling voicemail through a Microsoft Teams calling policy might also disable the voicemail service for your Skype for Business users.

Reviewer: ```suggestion
> In Skype for Business, when you disable voicemail with a Microsoft Teams calling policy, this may also disable the voicemail service for your Skype for Business users.
```
